source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1144,18 +1154,23 @@ def _get_cellCoordsText(self):
 			rowText=self.rowNumber
 		return "%s %s"%(colText,rowText)
 
-	def _get_columnNumber(self):
+	def _get_presentationalColumnNumber(self):
 		index=self.IA2Attributes.get('colindex')
+		try:

```

Comment:
`_get_presentationalRowNumber` also tries the parent, where this doesn't. Why is it different?
The rest of this code looks the same, I suggest extracting it for re-use.

Reply:
We expect that row and column numbers are exposed on cells. But some implementations may only expose rowIndex on the row object (parent of the cells) itself. Tables never expose column objects though thus we never need to worry about that. 
I'm not sure abstracting the rest of that code makes it any more readable. I'm struggling to work out what you would actually call it.